### PR TITLE
ukbd: Enable ukbd_apple_swap for all Apple ISO keyboards

### PR DESCRIPTION
**Pull request by frarohi**
Key code swapping between [<>] and [^°] key is enabled for all Apple ISO type keyboards. Before, swapping was enabled when the Eject key was detected in HID usage. This did not correlate well with the swapped keys presence.

usbdevs file is extended by several Apple keyboard models to support ISO model identification.

### CODE REVIEW

File: sys/dev/usb/input/ukbd.c [hunk 1]
```diff
@@ -1138,6 +1187,17 @@ ukbd_attach(device_t dev)
 		sc->sc_fkeymap[n] = fkey_tab[n];
 	}
 
+	/* check if this is an Apple keyboard with swapped key codes
+	 * apparently, these are the ISO layout models
+	*/
+	DPRINTF("uaa vendor: 0x%04x, uaa product 0x%04x\n", uaa->info.idVendor, uaa->info.idProduct );
+	if ((usbd_lookup_id_by_uaa(ukbd_apple_iso_models, sizeof(ukbd_apple_iso_models), uaa)) == 0) {
```
Comment (markjdb) [inline sys/dev/usb/input/ukbd.c:1194]: There are extra parens around the function call.

File: sys/dev/usb/input/ukbd.c [hunk 2]
```diff
@@ -173,13 +175,19 @@ struct ukbd_softc {
...
-#define	UKBD_FLAG_APPLE_SWAP	0x00000100
-#define	UKBD_FLAG_NUMLOCK	0x00080000
-#define	UKBD_FLAG_CAPSLOCK	0x00100000
+/* set in ukbd_attach */
+#define	UKBD_FLAG_APPLE_SWAP	0x00000040
+/* set in ukbd_parse_hid */
+#define	UKBD_FLAG_APPLE_EJECT	0x00000080
+#define	UKBD_FLAG_APPLE_FN		0x00000100
+#define	UKBD_FLAG_NUMLOCK		0x00080000
+#define	UKBD_FLAG_CAPSLOCK		0x00100000
```
Comment (markjdb) [inline sys/dev/usb/input/ukbd.c:184]: The whitespace here looks inconsistent.

File: sys/dev/usb/usbdevs [hunk 3]
```diff
@@ -1193,6 +1225,17 @@ product APPLE WELLSPRING8_JIS	0x0292	Apple Internal Keyboard/Trackpad
 product APPLE WELLSPRING9_ANSI	0x0272	Apple Internal Keyboard/Trackpad
 product APPLE WELLSPRING9_ISO	0x0273	Apple Internal Keyboard/Trackpad
 product APPLE WELLSPRING9_JIS	0x0274	Apple Internal Keyboard/Trackpad
+/**/
```
Comment (markjdb) [inline sys/dev/usb/usbdevs:1228]: Missing comment body here?

File: sys/dev/usb/input/ukbd.c [hunk 4]
```diff
@@ -1169,7 +1229,7 @@ ukbd_attach(device_t dev)
 	    (err != 0) || ukbd_any_key_valid(sc) == false) {
 		DPRINTF("Forcing boot protocol\n");
 
-		err = usbd_req_set_protocol(sc->sc_udev, NULL, 
+		err = usbd_req_set_protocol(sc->sc_udev, NULL,
```
Comment (markjdb) [inline sys/dev/usb/input/ukbd.c:1232]: Please don't make unrelated whitespace changes.

**Review by markjdb — COMMENTED** [review]
This seems fine to me, I just pointed out a few nits.

File: sys/dev/usb/input/ukbd.c (hunk 1 above)
Comment (frarohi) [reply sys/dev/usb/input/ukbd.c]: done, removed

File: sys/dev/usb/input/ukbd.c (hunk 2 above)
Comment (frarohi) [reply sys/dev/usb/input/ukbd.c]: adjusted (and changed tab size in my editor)

File: sys/dev/usb/usbdevs (hunk 3 above)
Comment (frarohi) [reply sys/dev/usb/usbdevs]: done, removed

File: sys/dev/usb/input/ukbd.c (hunk 4 above)
Comment (frarohi) [reply sys/dev/usb/input/ukbd.c]: reverted

**Review by markjdb — APPROVED** [review]
Looks good to me, thanks.

**Comment by markjdb** [comment]
Thanks for the patch.
